lib/msal-node/src/cache/distributed/IPartitionManager.ts

```diff
@@ -9,32 +9,27 @@ export interface IPartitionManager {
     /**
      * This function should return the correct key from which to read
      * the specific user's information from cache.
-     * 
+     *
```

Comment:
These changes appear to be unrelated. It's fine for this PR but let's try to break these into separate PRs in the future.

Reply:
Oh, ok :/ -just so that I understand correctly, the change here was to remove the link to this sample in the comments, you still think it's more apt for these two be done in separate PRs?

Reply:
Ah sorry, missed the part where you removed that, all good. But that's a perfect example of how too many formatting changes can distract from the more meaningful changes 😉

Reply:
Got it! I think we could use a sort of vscode extension to sync everyone's editor's formatting settings -maybe something to look into in the future : ))